tests/gtest/are_images_equal.cc

```diff
@@ -11,9 +11,9 @@
 using libavif::testutil::AvifImagePtr;
 
 int main(int argc, char** argv) {
-  if (argc != 4) {
+  if (argc != 4 && argc != 5) {
```

Comment:
Optional: I wonder if we should add a new program called `are_images_similar`.

Reply:
@vrabaud @y-guyon Vincent, Yannis: A heads-up since you are the main users of `are_images_equal`. This pull request adds a new mode to the `are_images_equal` command: to check if the two images are similar.